src/content/h1/tools/hek-plus/readme.md

```diff
@@ -0,0 +1,34 @@
+**Halo Editing Kit Plus (HEK+)** is a tool for extracting tags from Halo Custom
+Edition maps, as well as "protecting" maps to prevent tag extraction.
```

Comment:
The map cache file page has a section about protected maps and why they’re bad that can be linked to from here.